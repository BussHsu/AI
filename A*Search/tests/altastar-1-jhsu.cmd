--energy 1099 --start-x=1 --start-y=0
