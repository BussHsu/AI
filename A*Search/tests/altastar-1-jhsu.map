66 88 99
66 77 1
55 44 1
22 33 1
11 0  1
