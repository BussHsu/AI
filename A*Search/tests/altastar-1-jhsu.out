Solution steps: ['W', 'N', 'E', 'N', 'W', 'N', 'E', 'N', 'E']
Solution cost: 1098
Number of states considered: 12

Frontier:
Pos=(2, 3) Moves=['W', 'N', 'E', 'N', 'W', 'N', 'E', 'E'] Cost=931
Pos=(2, 2) Moves=['W', 'N', 'E', 'N', 'E'] Cost=532
Pos=(2, 1) Moves=['E', 'N'] Cost=3

Closed List:
Pos=(1, 0) Moves=[] Cost=0
Pos=(0, 0) Moves=['W'] Cost=122
Pos=(0, 1) Moves=['W', 'N'] Cost=244
Pos=(1, 1) Moves=['W', 'N', 'E'] Cost=366
Pos=(2, 0) Moves=['E'] Cost=2
Pos=(1, 2) Moves=['W', 'N', 'E', 'N'] Cost=488
Pos=(0, 2) Moves=['W', 'N', 'E', 'N', 'W'] Cost=610
Pos=(0, 3) Moves=['W', 'N', 'E', 'N', 'W', 'N'] Cost=732
Pos=(0, 4) Moves=['W', 'N', 'E', 'N', 'W', 'N', 'N'] Cost=733
Pos=(1, 3) Moves=['W', 'N', 'E', 'N', 'W', 'N', 'E'] Cost=854
Pos=(1, 4) Moves=['W', 'N', 'E', 'N', 'W', 'N', 'E', 'N'] Cost=976
Pos=(2, 4) Moves=['W', 'N', 'E', 'N', 'W', 'N', 'E', 'N', 'E'] Cost=1098
